App/iOS/Info.plist

```diff
@@ -82,6 +82,83 @@
 			<false/>
 		</dict>
 	</dict>
+	<key>CFBundleIcons~ipad</key>
+    <dict>
+        <key>CFBundleAlternateIcons</key>
+        <dict>
+            <key>icon-1</key>
+            <dict>
+                <key>CFBundleIconFiles</key>
+                <array>
+                    <string>icon-1</string>
+                </array>
+            </dict>
+            <key>icon-2</key>
+            <dict>
+                <key>CFBundleIconFiles</key>
+                <array>
+                    <string>icon-2</string>
+                </array>
+            </dict>
+            <key>icon-3</key>
+            <dict>
+                <key>CFBundleIconFiles</key>
+                <array>
+                    <string>icon-3</string>
+                </array>
+            </dict>
+            <key>icon-4</key>
+            <dict>
+                <key>CFBundleIconFiles</key>
+                <array>
+                    <string>icon-4</string>
+                </array>
+            </dict>
+            <key>icon-5</key>
+            <dict>
+                <key>CFBundleIconFiles</key>
+                <array>
+                    <string>icon-5</string>
+                </array>
+            </dict>
+            <key>icon-6</key>
+            <dict>
+                <key>CFBundleIconFiles</key>
+                <array>
+                    <string>icon-6</string>
+                </array>
+            </dict>
+            <key>icon-7</key>
+            <dict>
+                <key>CFBundleIconFiles</key>
+                <array>
+                    <string>icon-7</string>
+                </array>
+            </dict>
+            <key>icon-8</key>
+            <dict>
+                <key>CFBundleIconFiles</key>
+                <array>
+                    <string>icon-8</string>
+                </array>
+            </dict>
+            <key>icon-iso</key>
+            <dict>
+                <key>CFBundleIconFiles</key>
+                <array>
+                    <string>icon-iso</string>
+                </array>
+            </dict>
+        </dict>
+        <key>CFBundlePrimaryIcon</key>
+        <dict>
+            <key>CFBundleIconFiles</key>
+            <array/>
+            <key>UIPrerenderedIcon</key>
+            <false/>
+        </dict>
+    </dict>
+
```

Comment:
Good find, thanks for tracking down and PRing! Just wanna clean up the indent quick:
```suggestion
	<dict>
		<key>CFBundleAlternateIcons</key>
		<dict>
			<key>icon-1</key>
			<dict>
				<key>CFBundleIconFiles</key>
				<array>
					<string>icon-1</string>
				</array>
			</dict>
			<key>icon-2</key>
			<dict>
				<key>CFBundleIconFiles</key>
				<array>
					<string>icon-2</string>
				</array>
			</dict>
			<key>icon-3</key>
			<dict>
				<key>CFBundleIconFiles</key>
				<array>
					<string>icon-3</string>
				</array>
			</dict>
			<key>icon-4</key>
			<dict>
				<key>CFBundleIconFiles</key>
				<array>
					<string>icon-4</string>
				</array>
			</dict>
			<key>icon-5</key>
			<dict>
				<key>CFBundleIconFiles</key>
				<array>
					<string>icon-5</string>
				</array>
			</dict>
			<key>icon-6</key>
			<dict>
				<key>CFBundleIconFiles</key>
				<array>
					<string>icon-6</string>
				</array>
			</dict>
			<key>icon-7</key>
			<dict>
				<key>CFBundleIconFiles</key>
				<array>
					<string>icon-7</string>
				</array>
			</dict>
			<key>icon-8</key>
			<dict>
				<key>CFBundleIconFiles</key>
				<array>
					<string>icon-8</string>
				</array>
			</dict>
			<key>icon-iso</key>
			<dict>
				<key>CFBundleIconFiles</key>
				<array>
					<string>icon-iso</string>
				</array>
			</dict>
		</dict>
		<key>CFBundlePrimaryIcon</key>
		<dict>
			<key>CFBundleIconFiles</key>
			<array/>
			<key>UIPrerenderedIcon</key>
			<false/>
		</dict>
	</dict>
```